src/Plugin.php

```diff
@@ -234,6 +239,35 @@ public static function xmlrpc_methods($methods) {
     return $methods;
   }
 
+  /**
+   * @implements wp_head
+   */
+  public static function wp_head() {
+    $data = [
+      'ajaxurl' => admin_url('admin-ajax.php'),
+      'consent_version' => CORE_STANDARDS_COOKIE_CONSENT_VERSION,
+    ];
+    echo '<script>var core_standards = ' . json_encode($data) . ';</script>';
+  }
+
+  /**
+   * Logs cookie consent message.
+   */
+  public static function log_cookie_consent() {
+    $consent = $_POST['consent'] ?? [];
```

Comment:
The consent data should never be empty.  We should log an error instead and return early if it is.